mpas_analysis/ocean/nino34_index.py

```diff
@@ -0,0 +1,226 @@
+"""
+Computes NINO34 index and plots the time series and power spectra
+
+Author
+------
+Luke Van Roekel
+
+Last Modified
+-------------
+03/21/2017
+"""
+from ..shared.plot.plotting import nino34_timeseries_plot, nino34_spectra_plot
+import xarray as xr 
+import pandas as pd
+import numpy as np
+
+from ..shared.io import NameList, StreamsFile
+from ..shared.io.utility import buildConfigFullPath
+from ..shared.climatology import climatology
+from ..shared.constants import constants
+
+from ..shared.generalized_reader.generalized_reader \
+    import open_multifile_dataset
+
+from ..shared.timekeeping.utility import get_simulation_start_time, \
+    date_to_days, days_to_datetime
+
+# {{{
+def nino34_index(config, streamMap=None, variableMap=None):
```

Comment:
@vanroekel, here and elsewhere, the opening fold `# {{{` should go after the function definition (after the colon). This way, when someone closes the fold, they see the name and arguments of the function but not the contents.  The end fold goes either at the end of the last line or on its own line below the last, depending on how long the last line is.